docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/normalization/rms-norm-14.rst

```diff
@@ -0,0 +1,95 @@
+.. {#openvino_docs_ops_normalization_RMS_14}
+
+RMS
+===
+
+
+.. meta::
+  :description: Learn about RMS-14 - a normalization operation.
+
+**Versioned name**: *RMS-14*
+
+**Category**: *Normalization*
+
+**Short description**: Calculates Root Mean Square (RMS) normalization of the input tensor.
+
+**Detailed description**
+
+*RMSNorm* operation performs Root Mean Square (RMS) normalization on a given input ``data`` along dimensions specified by ``axes`` input.
+`Reference <https://arxiv.org/abs/1910.07467>`__.
+
+.. math::
+
+    (x / Sqrt(ReduceMean(x^2, axes) + eps))
+
+
+ -   If the optional ``scale`` input is provided:
+
+.. math::
+
+    (x / Sqrt(ReduceMean(x^2, axes) + eps)) * scale
+
+
+**Attributes**
+
+* *epsilon*
+
+  * **Description**: A very small value added to the variance for numerical stability. Ensures that division by zero does not occur for any normalized element.
+  * **Range of values**: a positive floating-point number
+  * **Type**: ``float``
+  * **Required**: *yes*
+
+* *compute_type*
+
+  * **Description**: The precision for internal computation, before scaling.
+  * **Range of values**: Supported floating point type: "f32", "f16", ...
```

Comment:
any other types except `fp16` and `fp32`?

Reply:
In the models I've seen cast to f32, in general any type can be allowed to comply with Convert capabilities, but it can be not a real use case.